packages/web-components/src/_docs/getting-started/overview.stories.mdx

```diff
@@ -2,19 +2,21 @@ import { Meta } from '@storybook/addon-docs';
 
 <Meta title="Getting Started/Overview" />
 
-## Overview
+## The official Fluent UI Web Component library
 
-## What are the Fluent UI Web Components?
+###
```

Comment:
@brookdozer do we need this paragraph here on line 7 if there isn't any content?

```suggestion
```

Reply:
Nope. Removed